juniper_official/routing/collect-isis-counters.rule

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
2.2.0 is very close. Which release are we targeting these features?

Reply:
we are targeting for 2.3.0 release